src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -248,6 +253,57 @@ public ITextComponent getShortDisplayString()
         }
     }
 
+    public static class SmeltAbleOreRequest extends AbstractRequest<SmeltableOre>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 43 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)